photon-core/src/main/java/org/photonvision/common/hardware/Platform.java

```diff
@@ -89,6 +106,10 @@ public static String getPlatformName() {
         }
     }
 
+    public static final String getNativeLibraryFolderName() {
```

Comment:
`static final` is redundant, should be just `static`

Reply:
Final is not redundant?

Reply:
I belive that if a method is static it is already final.